SimCalorimetry/EcalSimAlgos/interface/EcalHitResponse.h

```diff
@@ -33,12 +33,12 @@ class EcalHitResponse {
 
   typedef std::unordered_map<uint32_t, double> CalibCache;
 
-  enum { BUNCHSPACE = 25 };
-
   EcalHitResponse(const CaloVSimParameterMap* parameterMap, const CaloVShape* shape);
 
   virtual ~EcalHitResponse();
 
+  const float kSamplePeriod = 25;
```

Comment:
`Samp_Period` is a constant defined in EcalConstants.h. Better use that one.

Reply:
indeed it is used in the EBHitResponse, which override EcalHitResponse. But I can change it, sure